po/R-ca.po

```diff
@@ -0,0 +1,293 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: pkgdown 2.0.6.9000\n"
+"POT-Creation-Date: 2022-11-25 22:59+0100\n"
+"PO-Revision-Date: 2023-08-16 00:23+0200\n"
+"Last-Translator: jmaspons <joanmaspons@gmail.com>\n"
+"Language: ca\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Lokalize 23.04.3\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "No s'ha trobat la pàgina (404)"
+
+#: build-404.R:12
+msgid "Content not found. Please use links in the navbar."
+msgstr ""
+"No s'ha trobat el contingut. Useu els enllaços de la barra de navegació."
+
+#: build-articles.R:381 navbar.R:136 navbar.R:143 navbar.R:158
+msgid "Articles"
+msgstr "Articles"
+
+#: build-articles.R:430
+msgid "All vignettes"
+msgstr "Totes les vinyetes"
+
+#: build-footer.R:23
+msgid "Developed by"
+msgstr "Desenvolupat per"
+
+#: build-footer.R:28
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Lloc construït amb <a href=\"%s\">pkgdown</a> %s."
+
+#: build-home-authors.R:69
+msgid "More about authors..."
+msgstr "Més sobre els autors..."
+
+#: build-home-authors.R:72
+msgid "Developers"
+msgstr "Desenvolupadors"
```

Comment:
@jmaspons @llrs a bit late but I was wondering if those were gender-inclusive terms? Example in Spanishhttps://github.com/r-lib/pkgdown/blob/d5432a4e04655c6f9903196890f6bf166fbbd86b/po/R-es.po#L58

Reply:
Oh right, we can improve this using gender-neutral forms following https://www.softcatala.org/guia-estil-de-softcatala/aspectes-linguistics/ > Neutralització del gènere

I will review the whole file. Thanks for the heads-up, @maelle

Reply:
thank you!! and sorry to not have mentioned this earlier.

Reply:
I'm starting the review. In some cases, and as Softcatalà discourages doubling the forms, I would use the name of the action instead of the name of the person doing it (eg. autor -> autoria, traductor -> traducció). Is that OK, @llrs @maelle ?

Reply:
It wouldn't be right for me to have any opinion as it's not "my" language :wink: